cli/flox-activations/src/activations.rs

```diff
@@ -38,20 +37,23 @@ impl Activations {
     /// Get a mutable reference to the activation with the given ID.
     ///
     /// Used internally to manipulate the state of an activation.
-    pub fn activation_for_id_mut(&mut self, activation_id: Uuid) -> Option<&mut Activation> {
+    pub fn activation_for_id_mut(
+        &mut self,
+        activation_id: impl AsRef<str>,
```

Comment:
nit: similarly to my comment below, i think `AsRef<str>` provides slightly too large boundaries as to what is expected to be an activation Id, e.g. this method assumes that "anything that can be referenced as a string" can be an id, which is not quite true.